Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoPostgresSettingsUtil.java

```diff
@@ -0,0 +1,177 @@
+        try {
+            muConn = UserPreferences.getDatabaseConnectionInfo();
+        } catch (UserPreferencesException ex) {
+            LOGGER.log(Level.SEVERE, "Failed to import settings from multi-user settings.", ex);
+            return settings;
+        }
+        
+        handleTry(() -> settings.setHost(valOrDefault(muConn.getHost(), DEFAULT_HOST)));
+        handleTry(() -> settings.setDbName(DEFAULT_DBNAME));
+        handleTry(() -> settings.setUserName(valOrDefault(muConn.getUserName(), DEFAULT_USERNAME)));
+        
+        handleTry(() -> settings.setPort(valOrDefault(muConn.getPort(), DEFAULT_PORT, 1, 65535)));
+        handleTry(() -> settings.setBulkThreshold(RdbmsCentralRepo.DEFAULT_BULK_THRESHHOLD));
+        
+        handleTry(() -> settings.setPassword(valOrDefault(muConn.getPassword(), DEFAULT_PASSWORD)));
+        
+        return settings;
+    }
+    
+    
+    public static PostgresConnectionSettings loadCustomSettings() {
+        PostgresConnectionSettings settings = new PostgresConnectionSettings();
+        Map<String, String> keyVals = ModuleSettings.getConfigSettings(MODULE_KEY);
+        
+        
+        handleTry(() -> settings.setHost(valOrDefault(keyVals.get(HOST_KEY), DEFAULT_HOST)));
+        handleTry(() -> settings.setDbName(valOrDefault(keyVals.get(DBNAME_KEY), DEFAULT_DBNAME)));
+        handleTry(() -> settings.setUserName(valOrDefault(keyVals.get(USER_KEY), DEFAULT_USERNAME)));
+        
+        handleTry(() -> settings.setPort(valOrDefault(keyVals.get(PORT_KEY), DEFAULT_PORT, 1, 65535)));
+        handleTry(() -> settings.setBulkThreshold(valOrDefault(keyVals.get(BULK_THRESHOLD_KEY), RdbmsCentralRepo.DEFAULT_BULK_THRESHHOLD, 1, null)));
+        
+        String passwordHex = keyVals.get(PASSWORD_KEY);
+        String password;
+        try {
+            password = TextConverter.convertHexTextToText(passwordHex);
+        } catch (TextConverterException ex) {
+            LOGGER.log(Level.WARNING, "Failed to convert password from hex text to text.", ex);
+            password = DEFAULT_PASSWORD;
+        }
+        
+        final String finalPassword = password;
+        
+        handleTry(() -> settings.setPassword(finalPassword));
+        return settings;
+    }
+
+    public static void saveCustomSettings(PostgresConnectionSettings settings) {
+        Map<String, String> map = new HashMap<String, String>();
+        map.put(HOST_KEY, settings.getHost());
+        map.put(PORT_KEY, Integer.toString(settings.getPort()));
+        map.put(DBNAME_KEY, settings.getDbName());
+        map.put(BULK_THRESHOLD_KEY, Integer.toString(settings.getBulkThreshold()));
+        map.put(USER_KEY, settings.getUserName());
+        try {
+            map.put(PASSWORD_KEY, TextConverter.convertTextToHexText(settings.getPassword())); // NON-NLS
+        } catch (TextConverterException ex) {
+            LOGGER.log(Level.SEVERE, "Failed to convert password from text to hex text.", ex);
```

Comment:
In the previous section this is a warning and the default password is used. Why is it different here? Also if the error is "SEVERE" we probably shouldn't just log it and continue on.